rust/tools/authorization-logic/src/signing/mod.rs

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 The Raksha Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+pub mod export_assertions;
+pub mod import_assertions;
+
+// the only reason tink is not encapsulated is that
```

Comment:
nit: Comment should start with a capital.

Reply:
Fixed.